### PR TITLE
upd(prismlauncher-git): `7.2` -> `8.2`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

File: packages/prismlauncher-git/prismlauncher-git.pacscript
Comment (vigress8) [inline packages/prismlauncher-git/prismlauncher-git.pacscript:1]: Pacstall clones with submodules already, so you can remove that `prepare()`. It's left from the olden days

**Comment by Xdavius** [comment]
I'm stuck with the new packages that Debian are actually changing in Sid.

I don't know what to to for the moment and Sid is completely dead at the moment with Qt changements...

Can be merged and we need to wait for Sid reborn to get a look with the new build dépendancies name

**Comment by Xdavius** [comment]
```E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/noble/universe/binary-arm64/Packages  404  Not Found [IP: 91.189.91.83 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.
Error: Process completed with exit code 100.```

I don't understand this error on ubuntu:devel, master

**Comment by Elsie19** [comment]
Any updates @Xdavius?

**Comment by Xdavius** [comment]
> Any updates @Xdavius?

Will pass it to 8.2 today.
And hope nothing broken anymore...

**Pull request merged**